inline_test.go

```diff
@@ -72,7 +72,12 @@ func doTestsInlineParam(t *testing.T, tests []string, extensions, htmlFlags int,
 		input := tests[i]
 		candidate = input
 		expected := tests[i+1]
+		expected = strings.Replace(expected, "\r\n", "\n", -1)
+		expected = strings.Replace(expected, "\r", "\n", -1)
```

Comment:
Line ending conventions are not relevant here, since the test cases are
fully specified in the repository files. Nothing is being generated by an
external tool where line endings might vary by host OS.

On Sat, Feb 14, 2015 at 9:00 PM, elian0211 <notifications@github.com> wrote:

> In inline_test.go
> <https://github.com/russross/blackfriday/pull/151#discussion_r24719507>:
>
> > @@ -72,7 +72,12 @@ func doTestsInlineParam(t *testing.T, tests []string, extensions, htmlFlags int,
> >  		input := tests[i]
> >  		candidate = input
> >  		expected := tests[i+1]
> > +		expected = strings.Replace(expected, "\r\n", "\n", -1)
> > +		expected = strings.Replace(expected, "\r", "\n", -1)
>
> A newline on Windows equals "\r\n"
> on Mac OS : "\r"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/russross/blackfriday/pull/151/files#r24719507>.
>